packages/pluggableWidgets/tree-view-web/src/TreeView.tsx

```diff
@@ -0,0 +1,25 @@
+import { createElement, FunctionComponent } from "react";
+import { TreeViewContainerProps } from "../typings/TreeViewProps";
+import { TreeView as TreeViewComponent } from "./components/TreeView";
+
+export const TreeView: FunctionComponent<TreeViewContainerProps> = props => {
```

Comment:
I think I used it at certain places so I only need to type the function as a whole once through `FunctionComponent`, rather than typing the props and return element separately. But I can change this.

Reply:
Fixed